server/accounts.go

```diff
@@ -2623,6 +2629,35 @@ func (s *Server) UpdateAccountClaims(a *Account, ac *jwt.AccountClaims) {
 			}
 		}
 	}
+
+	// if we encountered issues during this process, retry in a bit
+	if incomplete {
+		accName := old.Name
+		time.AfterFunc(time.Second, func() {
+			s.reloadAccount(accName)
+		})
+	}
+}
+
+func (s *Server) reloadAccount(accName string) {
+	if !s.isRunning() {
+		return
+	}
+	if acc, ok := s.accounts.Load(accName); ok {
+		acc := acc.(*Account)
```

Comment:
Usually like to use diff variable name vs aliasing.

Reply:
changed first acc to a